tests/modules/encounters/resources/test_delete_encounter.py

```diff
@@ -24,10 +24,7 @@ def test_delete_method(db, flask_app_client, researcher_1, test_root, staff_user
             {'locationId': 'test2'},
         ],
     }
-    response = sighting_utils.create_sighting(
-        flask_app_client, researcher_1, expected_status_code=200, data_in=data_in
-    )
-    assert response.json['success']
+    response = sighting_utils.create_sighting(flask_app_client, researcher_1, data_in)
```

Comment:
Why is ensuring the sighting was created removed? Because it isn't the primary purpose of the test?